components/core/tools/scripts/lib_install/openssl.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+cUsage="Usage: ${BASH_SOURCE[0]} <version> [prefix] [openssldir]"
+if [ "$#" -lt 1 ] ; then
+    echo $cUsage
+    exit
+fi
+version="$1"
+prefix="${2:-/opt/openssl-${version}}"
+openssldir="${3:-$prefix/ssl}"  # Default to prefix/ssl if not specified
+
+package_name=openssl-${version}
+temp_dir="/tmp/${package_name}-installation"
+mkdir -p "$temp_dir"
+
+echo "🔧 OpenSSL version: $version"
+echo "📁 Install prefix: $prefix"
+echo "📂 OpenSSL directory: $openssldir"
```

Comment:
```suggestion
echo "OpenSSL version: ${version}"
echo "Install prefix: ${prefix}"
echo "OpenSSL directory: ${openssldir}"
```
- As discussed offline, let's remove these emoji
- Let's use `{}` for varaibles